dequorum/urls.py

```diff
@@ -5,6 +5,5 @@
 urlpatterns = [
     url(r'^$', views.thread_list, name='thread-list'),
     url(r'^add/$', views.thread_create, name='thread-create'),
-    url(r'^(?P<thread_pk>\d+)/$', views.thread_detail, name='thread-detail'),
```

Comment:
Leave trailing commas in future.

We wasted a minute trying to figure what you'd changed on this line -- NOTHING.